index.mdx

```diff
@@ -38,6 +38,37 @@ export default LandingContainer;
   <Header2>Explore open source projects currently in development.</Header2>
 </Jumbotron>
 
+<Box p={{ _: "8px", sm: "12px", md: "16px", lg: "24px" }}>
+  <FlexSection justifyContent="space-around" flexWrap="wrap">
+    <WideTile 
+      style={{ height: "284px"}}
+      to="docs/axelar/intro-to-axelar.md"
```

Comment:
In the future, we will have another dropdown within the Axelar section for "Axelar - XRPL EVM Sidechain Integration". No action is needed now.